src/lambda/oembed/oembed.js

```diff
@@ -34,7 +34,8 @@ function handler(event, context, callback) {
     );
   }
 
-  const { url, referrer, maxwidth = 900, maxheight = 300 } = params;
+  const { url, referrer, maxwidth = '100%', maxheight = 300 } = params;
```

Comment:
https://github.com/testing-library/testing-playground/pull/56#discussion_r432099970

Reply:
Totally forgot about that one 😓